ui/src/components/overview/MobileClientListViewItem.js

```diff
@@ -146,31 +102,48 @@ class MobileClientOverviewList extends Component {
                 stacked={false}
                 hideCloseIcon={true}
             >
-                <Row>
-                    <Col md={12}>
-                        {this.alert()}
-                    </Col>
-                </Row>
-                <Row>
-                    <Col md={6}>
-                        <ComponentSectionLabel>Mobile Services</ComponentSectionLabel>
-                        <MobileClientServiceChart mobileServices={mobileServices}></MobileClientServiceChart>
-                        <a>View All Mobile Services</a>
-                    </Col>
-                    <Col md={6}>
-                        <ComponentSectionLabel>Client Info</ComponentSectionLabel>
-                    </Col>
-                </Row>
-                <Row>
-                    <Col md={12}>
-                        <ComponentSectionLabel>Mobile Builds</ComponentSectionLabel>
-                        <MobileClientBuildList mobileClientBuilds={mobileClientBuilds}></MobileClientBuildList>
-                        <a>View All Mobile Builds</a>
-                    </Col>
-                </Row>
+            {
+                mobileServiceInstances.length ?
```

Comment:
would it be neater to put each fragment in a function to make this section more readable?